frontend/components/person-card/index.js

```diff
@@ -17,7 +17,12 @@ import { Heading, HEADING_OPTIONS } from '../shared/Heading';
  */
 
 const PersonCard = ({ cardDetails, bgColord, rounded, boxShadow }) => {
-  const { image, title, subtitle, description } = cardDetails;
+  const {
+    card_image: { url },
+    title,
+    sub_title,
+    descripe
```

Comment:
This `descripe`  didn't appear because the API is `describe` 
Just change the spelling and it will be very good
then change the styles to match the design